app/models/concerns/questionnaires/validations.rb

```diff
@@ -10,11 +10,26 @@ module Questionnaires::Validations
     validates :name, :email_subject, :email_link, length: { maximum: 255 },
       allow_nil: true, allow_blank: true
 
+    before_validation :normalize_pollable_type
+
+    validate  :pollable_type_cannot_be_cleared_if_used
+    validates :pollable_type, inclusion: { in: Questionnaire::POLLABLE_TYPES },
+      allow_nil: true
     before_destroy :check_for_answered_polls, prepend: true
   end
 
   private
 
+    def normalize_pollable_type
+      self.pollable_type = nil if pollable_type.blank?
+    end
+
+    def pollable_type_cannot_be_cleared_if_used
+      if pollable_type_was.present? && pollable_type.blank? && used_in_conclusion_reviews?
+        errors.add(:pollable_type, :cannot_be_cleared_if_used)
```

Comment:
```suggestion
        errors.add(:pollable_type, :taken)
```
Le pondría uno "estándar" y de última le cambiamos el mensaje (aunque el "Ya está en uso" _pega_, más que esto es casi imposible que se vea si lo limitamos desde el UI).